src/test/java/base/BaseTest.java

```diff
@@ -220,6 +220,11 @@ public boolean isContainsTextInUrl(String text) {
         return currentUrl.contains(text);
     }
 
+    public void inputTextAndClickEnter (By by,String text) {
+
+        getDriver().findElement(by).sendKeys(text + Keys.ENTER);
+        }
```

Comment:
Наверное так лучше будет
```suggestion
    public void inputTextAndClickEnter (By by, String text) {
        getDriver().findElement(by).sendKeys(text + Keys.ENTER);
    }
```